tests/phpunit/phpunit.xml

```diff
@@ -5,6 +5,7 @@
     <php>
         <const name="BLT_ENV" value="ci"/>
         <const name="BLT_ALIAS" value="self"/>
+        <const name="BLT_MULTISITE_NAME" value="default,g"/>
```

Comment:
Why is this defined as a constant in the phpunit.xml rather than just hardcoded in the test?